pkg/daemon/daemon.go

```diff
@@ -1120,6 +1192,54 @@ func (dn *Daemon) checkStateOnFirstRun() error {
 	return dn.triggerUpdateWithMachineConfig(state.currentConfig, state.desiredConfig)
 }
 
+func forceFileExists() bool {
+	_, err := os.Stat(constants.MachineConfigDaemonForceFile)
+
+	// No error means we could stat the file; it exists
+	if err == nil {
+		return true
+	}
+
+	// We know the file doesn't exist
+	if goErrors.Is(err, os.ErrNotExist) {
+		return false
+	}
+
+	// We couldn't stat the file for other reasons, assume it doesn't exist
```

Comment:
Hmm, maybe we should at least log something here. I guess we technically never checked for other errors, so this is fine as is as well